data/campaigns/Two_Brothers/_main.cfg

```diff
@@ -18,7 +18,9 @@
     {CAMPAIGN_DIFFICULTY EASY "units/human-loyalists/horseman/horseman.png~RC(magenta>red)" ( _ "Horseman") ( _ "Beginner")} {DEFAULT_DIFFICULTY}
     {CAMPAIGN_DIFFICULTY HARD "units/human-loyalists/grand-knight/grand-knight.png~RC(magenta>red)" ( _ "Grand Knight") ( _ "Challenging")}
 
-    description= _ "An evil mage is threatening the small village of Maghre and its inhabitants. The village’s mage sends to his warrior brother for help, but not all goes as planned. Can you help?
+    description= _ "An evil mage is threatening the small village of Maghre and its inhabitants. The village’s mage sends to his warrior brother for help, but not all goes as planned. Can you help?" + "
+
+" + _ "This campaign has two difficulties, with a large difference between them. The Grand Knight difficulty is intended for players who’ve already completed Liberty and Heir to the Throne." + "
```

Comment:
Can you make this smaller and more grey (color) than the main description?

Reply:
I made it smaller, but disagree about making it greyer, at least for the moment, because it's already on a greyish background, and the title screen redesign might put it on a transparent background. As long as the text is there to be translated, we can change the color after the start of the 1.20 string freeze.